src/lib/resource/sections/context-pregnancy.js

```diff
@@ -18,9 +18,9 @@ module.exports = {
                     resources: {
                         'pregnancy-context': {
                             description:
-                                '<p class="govuk-body">We need to ask questions about pregnancy related to the crime. These questions may be difficult to answer or may not be relevant to you.</p><p class="govuk-body">By answering them, it ensures you are considered for all correct compensation from us. This helps us better understand the case and if we need to ask for more evidence from you.</p><h2 class="govuk-heading-m">Help and support</h2><p class="govuk-body">You can get practical and emotional support after a crime. Visit <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">Victim and Witness information (opens in new tab)</a> if you’re in England and Wales. Or <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">support as a victim or witness of crime (opens in new tab)</a> in Scotland.</p>{% set templateHtml %}{% include \'contact.njk\' %}{% endset %}{{ govukDetails({summaryText: "If you need help or support with the application",html: \'<p class="govuk-body">You can contact us for help with your application.</p>\' + templateHtml + \'<p class="govuk-body">You can <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">get practical or emotional support (opens in new tab)</a> after a crime.</p><p class="govuk-body">There is different practical or emotional support <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">if you live in Scotland (opens in new tab)</a>.</p>\'})}}',
+                                '<p class="govuk-body">We need to ask questions about pregnancy related to the crime. These questions may be difficult to answer or may not be relevant to you.</p><p class="govuk-body">By answering them, it ensures you are considered for all correct compensation from us. This helps us better understand the case and if we need to ask for more evidence from you.</p><h2 class="govuk-heading-m">Help and support</h2><p class="govuk-body">The following links open in a new tab.</p><p class="govuk-body">You can get practical and emotional support after a crime. Visit <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">Victim and Witness information</a> if you’re in England and Wales. Or <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">support as a victim or witness of crime</a> in Scotland.</p>',
```

Comment:
Should we adding ` (opens in new tab)` in the link tag if they have the `target="_blank"` attribute?

Reply:
@armoj that looks to have been replaced with _The following links open in a new tab._

Reply:
My eye's were drawn to the anchor tags. Didn't sport that text 🤝

Reply:
The [guidance ](https://design-system.service.gov.uk/styles/links/#opening-links-in-a-new-tab) does say that we should be adding some visually hidden text after the link for the use with screen readers:

> including `<span class="govuk-visually-hidden">(opens in new tab)</span>` as part of the link text, so that part of the link text is visually hidden but still accessible to screen readers

Reply:
Good shout!